Data-Structure/graphs/depthFirstTraversalOfGraph.c

```diff
@@ -0,0 +1,88 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+typedef struct graph{
+	int nVertices;   //number of vertices in the graph
+	int nEdges;      //number of edges in the graph
+	int **adjMatrix;
+	}Graph;
+	
+Graph * createAdjMatrixOfGraph(Graph *);	
+	
+/* this will create Adjacency matrix of the undirected graph graph */
+Graph * createAdjMatrixOfGraph(Graph *g){
+	
+	int i, u, v;
+	
+	g = (Graph *)malloc(sizeof(Graph));
+	
+	printf("Enter the number of Vertices and Edges:");
+	scanf("%d%d", &g->nVertices, &g->nEdges);
+	
+	g->adjMatrix = (int **)malloc(sizeof(int) * g->nVertices);
```

Comment:
yes, it should be. There would be some minor changes in BreadthFirstTraversing as well.